### PR TITLE
UI: Fix stats window geometry saving

### PR DESCRIPTION
**Pull request by SuslikV**
Possible fix for: https://obsproject.com/mantis/view.php?id=923

If you close whole application (not just Stats window) it skips the
close event and thus saving of the geometry.

### CODE REVIEW

File: UI/window-basic-main.cpp [hunk 1]
```diff
@@ -2938,7 +2938,7 @@ void OBSBasic::CloseDialogs()
 		projector.clear();
 	}
 
-	delete stats;
+	if (!stats.isNull()) stats->close(); //call close to save Stats geometry
```
Comment (Xaymar) [inline UI/window-basic-main.cpp:2941]: Why did you remove the delete?

Comment (SuslikV) [inline UI/window-basic-main.cpp]: I thought that Qt::WA_DeleteOnClose can do it automatically when event accepted...

**Comment by jp9000** [comment]
Thank you for the fix.  Fixing up the commit message then merging.  Make sure that the long description part of the commit message is no longer than 72 columns in the future.
